internal/backends/python/python.go

```diff
@@ -84,10 +85,15 @@ func normalizeSpec(spec interface{}) string {
 
 // normalizePackageName implements NormalizePackageName for the Python
 // backends.
+// See https://packaging.python.org/en/latest/specifications/name-normalization/
 func normalizePackageName(name api.PkgName) api.PkgName {
 	nameStr := string(name)
 	nameStr = strings.ToLower(nameStr)
-	nameStr = strings.Replace(nameStr, "_", "-", -1)
+	matcher, err := regexp.Compile(`[-_.]+`)
```

Comment:
We can use regexp.MustCompile and set a global variable at the module level to avoid re-compiling.

Reply:
Ah, good one! I'll make the change